token/js/src/extensions/group/actions.ts

```diff
@@ -0,0 +1,130 @@
+import type { ConfirmOptions, Connection, PublicKey, Signer, TransactionSignature } from '@solana/web3.js';
+import { sendAndConfirmTransaction, Transaction } from '@solana/web3.js';
+import {
+    createInitializeGroupInstruction,
+    createUpdateGroupMaxSizeInstruction,
+    createUpdateGroupAuthorityInstruction,
+} from '@solana/spl-token-group';
+
+import { TOKEN_2022_PROGRAM_ID } from '../../constants.js';
+import { getSigners } from '../../actions/internal.js';
+
+/**
+ * Initialize a new `Group`
+ *
+ * Assumes one has already initialized a mint for the group.
+ *
+ * @param connection       Connection to use
+ * @param payer            Payer of the transaction fees
+ * @param mint             Mint Account
+ * @param mintAuthority    Mint Authority
+ * @param updateAuthority  Update Authority
+ * @param maxSize          Maximum number of members in the group
+ * @param multiSigners     Signing accounts if `authority` is a multisig
+ * @param confirmOptions   Options for confirming the transaction
+ * @param programId        SPL Token program account
+ *
+ * @return Signature of the confirmed transaction
+ */
+export async function tokenGroupInitializeGroup(
```

Comment:
Can we also get some instructions for "with rent transfer"? Even though `TokenGroup` and `TokenGroupMember` are fixed-size extensions, they are initialized after the mint is initialized, so it needs to be reallocated and thus requires some additional lamports. Updating the max size and authority has no rent effect.